tests/port_dpb.py

```diff
@@ -313,3 +317,34 @@ def change_speed_and_verify(self, dvs, port_names, speed = 100000):
             p.verify_app_db()
             time.sleep(1)
             p.verify_asic_db()
+
+    def verify_port_breakout_mode(self, dvs, port_name, breakout_mode):
+        p = Port(dvs, port_name)
+        p.verify_breakout_mode(breakout_mode)
+
+    def get_child_ports(self, root_port, breakout_mode):
+
+        if '+' not in breakout_mode:
+            return self._get_child_ports(root_port, breakout_mode, self.MAX_LANES)
+
+        modes = breakout_mode.split('+')
+        child_ports = []
+        root_port_num = int(root_port.split('Ethernet')[1])
+        for mode in modes:
+            lanes = int(mode.split('(')[1].split(')')[0])
+            mode = mode.split('(')[0]
+            child_ports = child_ports + self._get_child_ports(root_port, mode, lanes)
+            root_port_num = root_port_num + lanes
+            root_port = 'Ethernet' + str(root_port_num)
```

Comment:
I think a short comment w/ an example of what the output looks like would be helpful for future maintainers